.github/workflows/spellchecker.yml

```diff
@@ -0,0 +1,21 @@
+name: spellchecker
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  misspell:
+    name: runner/misspell
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code.
+        uses: actions/checkout@v1
+      - name: Install
+        run: |
+          wget -O - -q https://git.io/misspell | sh -s -- -b .
+      - name: Misspell
+        run: |
+          find . -type f -maxdepth 1 | xargs ./misspell -error
+          find . -name "*.go" -type f | xargs ./misspell -error
+          find docs -type f | xargs ./misspell -error
+          find web/src web/cypress -type f | xargs ./misspell -error
```

Comment:
oh yes, how about checker for golang?

Reply:
What? I don't get your point.
`find . -name "*.go" -type f | xargs ./misspell -error` is checker for golang.

Reply:
got it 👏